blog_system/urls.py

```diff
@@ -0,0 +1,17 @@
+from django.conf import settings
+from django.conf.urls.static import static
+from django.contrib import admin
+from django.urls import path
+from django.urls.conf import include
+from debug_toolbar.toolbar import debug_toolbar_urls
```

Comment:
The debug_toolbar is a great tool for development, but it should not be included in production. Consider using a conditional statement to only include it when DEBUG is set to True.